app/src/main/java/com/asfoundation/wallet/ui/webview_payment/usecases/CreateWebViewPaymentOspUseCase.kt

```diff
@@ -35,7 +37,8 @@ class CreateWebViewPaymentOspUseCase @Inject constructor(
 
   operator fun invoke(
     transaction: TransactionBuilder,
-    appVersion: String?
+    appVersion: String?,
+    context: Context
```

Comment:
Context should not be used in the domain layer (use cases), otherwise the use case won't be testable.
It's better to pass the variable you need (for example hasCustomTab: Boolean) instead of the context.